generators/static.go

```diff
@@ -41,8 +55,49 @@ func (sg *StaticGenerator) Generate(component *core.Component) (manifest string,
 	return manifests, err
 }
 
-// Install is provided such that the StaticGenerator fulfills the Generator interface.
-// Currently is a no-op, but could be extended to support remote static content (see #155)
-func (sg *StaticGenerator) Install(component *core.Component) (err error) {
+// Install for StaticGenerator gives the ability to point to a single yaml
+// manifest over `method: http`; This is a noop for any all other methods.
+func (sg *StaticGenerator) Install(c *core.Component) (err error) {
+	if strings.EqualFold(c.Method, "http") {
```

Comment:
@edaena note: the logic for your previous remote yaml components has been moved here